tfx/scripts/tfx_test_installed.sh

```diff
@@ -113,7 +114,8 @@ SKIP_LIST+=(
 # xargs stops only when the exit code is 255, so we convert any
 # failure to exit code 255.
 
+# pytest is used now
+
 set -f  # Disable bash asterisk expansion.
-find src -name '*_test.py' \
-  ${SKIP_LIST[@]/#tfx/-not -path src} \
-  |  xargs -I {} sh -c "${PYTHON_BINARY} {} || exit 255"
+${PYTEST_BINARY} $(find src -name '*_test.py' \
+  ${SKIP_LIST[@]/#tfx/-not -path src} | tr '\n' ' ')
```

Comment:
I think we can get rid of this manual test collection - this is now pytest's job.